.github/workflows/Regression.yml

```diff
@@ -107,6 +107,89 @@ jobs:
       run: |
         python scripts/regression_test_runner.py --old=duckdb/build/release/benchmark/benchmark_runner --new=build/release/benchmark/benchmark_runner --benchmarks=.github/regression/imdb.csv --verbose --threads=2
 
+ regression-test-memory-safety:
+  name: Regression Tests between safe and unsafe builds
+  runs-on: ubuntu-20.04
+  env:
+    CC: gcc-10
+    CXX: g++-10
+    GEN: ninja
+    BUILD_BENCHMARK: 1
+    BUILD_TPCH: 1
+    BUILD_TPCDS: 1
+    BUILD_HTTPFS: 1
+    BUILD_JEMALLOC: 1
+
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        path: main
```

Comment:
```suggestion
```
I think this has to be taken out, so that on current folder there will be current (with checks enabled) and on unsafe/ the disabled check duckdb, and makes this job structure similar to the previous one.